Unescaped left brace in regex is illegal here in regex;
marked by <-- HERE in m/\${ <-- HERE ([^ \t=:+{}]+)}/ 
at /home/hdkiller/libre/LibreELEC.tv/build.LibreELEC-RPi2.arm-9.0-devel/toolchain/bin/automake-1.15 line 3936.

--- a/bin/automake.in_orig	2015-01-05 20:25:55.000000000 +0100
+++ b/bin/automake.in	2017-09-10 08:56:51.800291204 +0200
@@ -3878,7 +3878,7 @@ sub substitute_ac_subst_variables_worker
 sub substitute_ac_subst_variables
 {
   my ($text) = @_;
-  $text =~ s/\${([^ \t=:+{}]+)}/substitute_ac_subst_variables_worker ($1)/ge;
+  $text =~ s/\$\{([^ \t=:+{}]+)\}/substitute_ac_subst_variables_worker ($1)/ge;
   return $text;
 }
